brownie/network/transaction.py

```diff
@@ -775,9 +776,13 @@ def _expand_trace(self) -> None:
         last_map = {0: _get_last_map(self.receiver, self.input[:10])}  # type: ignore
         coverage_eval: Dict = {last_map[0]["name"]: {}}
 
+        datacopy_precompile = "0x0000000000000000000000000000000000000004"
```

Comment:
Why do you single out this specific precompile and not the others?

Reply:
Weak sauce on my part, changed now to use `re` and match against all the current pre-compiles as per geth's codebase